code/controller/actuator.h

```diff
@@ -1,13 +1,54 @@
 #ifndef ACTUATOR_H
 #define ACTUATOR_H
+
 #include "controller.h"
+#include "qextserialport.h"
+
+#define BYTE_RANGE	256
+#define DATA_SIZE	4
+#define CMD_SIZE	2
+#define STEP_FACTOR 5
+#define STEP_TIME	10
+
+// For Zaber, settings must be: 9600 baud, no hand shaking, 8 data bits, no parity, one stop bit.
+const PortSettings DEFAULT_SETTINGS = { BAUD9600, DATA_8, PAR_NONE, STOP_1, FLOW_XONXOFF, 10 };
+
+
+class QextSerialPort;
 
 
-class Actuator
+enum ZaberCmd {

```

Comment:
I will keep the same name, but add a comment. A name change here would be inappropriate since the commands are specific to the Zaber manufactured actuators.